<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisitCollegeWindow</class>
 <widget class="QDialog" name="VisitCollegeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="collegeNameLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>721</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>College Name</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="souvenirAndPriceTable">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>631</width>
     <height>381</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
   <property name="midLineWidth">
    <number>2</number>
   </property>
   <property name="rowCount">
    <number>7</number>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>Souvenir Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Price</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Quantity to Purchase</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="purchaseButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>510</y>
     <width>97</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Purchase</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
